tests/runtests.py

```diff
@@ -233,6 +234,22 @@ def actual_test_processes(parallel):
         return parallel
 
 
+class ActionSelenium(argparse.Action):
+    """
+    Validate the browsers requested and raise appropriate exception.
```

Comment:
the requested browsers
"and raise appropriate exception" can probably be considered part of validation.